src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_sysfs.c

```diff
@@ -386,8 +386,10 @@ static ssize_t shutdown_store(struct device *dev,
 	if (kstrtou32(buf, 10, &val) == -EINVAL)
 		return -EINVAL;
 
-	if (val)
-		xocl_queue_work(xdev, XOCL_WORK_SHUTDOWN, 0);
+	if (val == 1)
```

Comment:
Instead of using magic numbers (e.g., 1 or 2) is there a define value that can be used to represent these values instead.

Reply:
no. the magic numbers are already there, I just follow the existing way.
There are a lot other places having this coding style. Maybe a dedicate campaign is required to eliminate this, but not by this PR

Reply:
In the past coding style, the underlying meaning was 0 or non-zero, 0 means false. In your new code, 1 mean something else. If 1 has it's special meaning than 2 or 3, we'd better have an enum here.

Reply:
ok. will change